x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -52,6 +52,11 @@ export const allowedExperimentalValues = Object.freeze({
    */
   automatedProcessActionsEnabled: true,
 
+  /**
+   * Temporary feature flag to enable the Response Actions in Rules UI - intermediate release
+   */
+  rulesResponseActionsTemporaryFlagEnabled: false,
```

Comment:
nit: i'd suggest leaving out 'temporary' and 'flag', and giving a more descriptive name to the feature flag, e.g. `automatedResponseActionsForMoreRulesEnabled` or something